### ISSUE TITLE
Weird Annoying Resource pack bug

### ISSUE THREAD

**Issue by CharlieDowns** [Wynncraft/Issues#5248]
Hello, I want to report a weird resourcepack issue I had at random on all Wynncraft servers ingame when logging in from Hub.

When this happens, all blockmodels are enlarged (for example a huge annoying wand), also some textures of block and gear are missing/replaced by purple blocks.

The resourcepack downloaded from the server was not corrupted, I checked this.
I also experienced this on vanilla client, without mods like Optifine, Liteloader and other resourcepacks.

[Log](https://github.com/Wynncraft/Issues/files/319558/log.txt) of issue.

For reference: https://imgur.com/a/d7KRn

**Comment by thecoolkogut4** [comment]
are you in version 1.9 or 1.8? if 1.9 try 1.8. also if that wont work, go to .minecraft, open server-resource-pakcs and delete the legacy_wynnpack file. once you delete it, it will reinstall once you log back in. that should fix it

**Comment by onlylove943** [comment]
I have same issue. Pls let us know if u have way to fix

**Comment by CharlieDowns** [comment]
I primarily play on 1.9.4, but this also seems to have happened on 1.8.9.

Also researching the log file, I noticed "ResourceManager" loading a different server resourcepack.
The following resourcepack was loaded called 'c01b9763b0bc41deba9d4b74647da7fa74a444f7'.
Normally this should have been '0b88c016c0b811591eac6045e66c5fbadb64928c'.

I have reuploaded the resourcepack to TinyUpload "[c01b9763b0bc41deba9d4b74647da7fa74a444f7](http://s000.tinyupload.com/index.php?file_id=63610194487366514138)".
